### ISSUE TITLE
Export as SAT

### ISSUE THREAD

**Issue by andydandy74** [BadMonkeysInc/Dynanimator#12]
Make some nodes that export the current model state as a SAT file (this could be either the Revit model or the Dynamo model). There should be built-in nodes for this.

**Comment by jbenoit44** [comment]
I tried recently to make some export in 0.8, and it failed. quick try but failed anyway....

**Comment by andydandy74** [comment]
Yes, I've had problems with exporting larger / complex models as well...
